### DISCUSSION TITLE
Error in the Russian translation of the site.

### DISCUSSION THREAD

**Discussion by xTarazTwilightForever** [pcottle/learnGitBranching#980] [Q&A]
In the task "Introduction to cherry-pick". It is mentioned about copying changes from 3 commits to the "master" branch... Where did the "master" branch come from?", the illustration shows the "main" branch, in the original century "main", in all previous tasks there was a "main" branch, where did the "master" branch come from? 

Links to screenshots:
https://ibb.co/6gjv0sC
https://ibb.co/CBKVjpY

**Comment by xTarazTwilightForever** [comment]
At the "Select 1 commit" level, the same error occurs. 

https://ibb.co/ZNvzwfq

**Comment by pcottle** [comment]
> In the task "Introduction to cherry-pick". It is mentioned about copying changes from 3 commits to the "master" branch... Where did the "master" branch come from?", the illustration shows the "main" branch, in the original century "main", in all previous tasks there was a "main" branch, where did the "master" branch come from?

There's some more info on the rename here
<img width="922" alt="Screen Shot 2022-07-15 at 12 18 26 PM" src="https://user-images.githubusercontent.com/1135007/179286423-95c17608-b7ad-49e4-97fa-4d2d142be3d0.png">
https://github.com/github/renaming
